src/rdata/aaaa.rs

```diff
@@ -106,7 +111,7 @@ impl CanonicalOrd for Aaaa {
 
 impl RecordData for Aaaa {
     fn rtype(&self) -> Rtype {
-        Rtype::Aaaa
+        Aaaa::RTYPE
```

Comment:
I find this confusing. It looks like a namespace that is the wrong way around.

Reply:
Would it help if this were `Self::RTYPE`?

Reply:
Yes, I think that is better